test_python_install.py

```diff
@@ -0,0 +1,23 @@
+""" Run this file with nosetests to check your python installation.
```

Comment:
For consistency, this file should be called `check_env.py` and be runnable as a script. Adding an `if __name__ == "__main__"` clause that calls `nose.run()` should do the trick.